contracts/test/TestTablelandTablesUpgrade.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
```

Comment:
used to test storage upgrades and controller compatibility

Reply:
excellent